CHANGELOG.rst

```diff
@@ -179,6 +179,7 @@ Bugfixes
 - filesystem - improve error messages when output cannot be parsed by including newlines in escaped form (https://github.com/ansible-collections/community.general/pull/4700).
 - funcd connection plugin - fix signature of ``exec_command`` (https://github.com/ansible-collections/community.general/pull/5111).
 - ini_file - minor refactor fixing a python lint error (https://github.com/ansible-collections/community.general/pull/5307).
+- iso_create plugin - iso_create doesn't add folers (https://github.com/ansible-collections/community.general/pull/5468).
```

Comment:
Please remove this and the other changelog related changes, and instead create a file in `changelogs/fragments/` with the content you added to the YAML file as described [here](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment).

Reply:
Thanks. Done